NBitcoin/Transaction.cs

```diff
@@ -180,12 +180,15 @@ void SetNull()
 		{
 			return !(a == b);
 		}
+
 		public override bool Equals(object? obj)
 		{
-			var item = obj as OutPoint;
-			if (object.ReferenceEquals(null, item))
-				return false;
-			return item == this;
+			return Equals(obj as OutPoint);
+		}
+
+		public bool Equals(OutPoint? other)

```

Comment:
Is the `?` is some C#8 nullable thingy?

Reply:
Yes, `Equals` must accept `null`.

Reply:
Does your reply imply the answer to my question is yes?

Reply:
Yes.